llvm/test/Transforms/LoopVectorize/AArch64/veclib-function-calls-linear-ptrs.ll

```diff
@@ -0,0 +1,134 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "call.*(frexp|modf)" --version 4
```

Comment:
sincos (which also takes linear pointers) has tests inside veclib-function-calls.ll, so I would suggest to move this tests to the same file.